examples/fft_physical_x/fft_c2c_x.f90

```diff
@@ -97,8 +97,8 @@ program fft_c2c_x
    ph => decomp_2d_fft_get_ph()
    !  input is X-pencil data
    ! output is Z-pencil data
-   call alloc_x(in, ph, .true.)
-   call alloc_z(out, ph, .true.)
+   call alloc_x(in, ph, opt_global=.true.)
```

Comment:
When calling alloc subroutines, the argument `opt_global` will be mandatory after this PR ?

Reply:
I'll double check, in theory it shouldn't be. This could just have been an "apply everywhere" solution.